jax/scipy/optimize/bfgs_minimize.py

```diff
@@ -0,0 +1,127 @@
+"""The Broyden-Fletcher-Goldfarb-Shanno minimization algorithm.
+https://pages.mtu.edu/~struther/Courses/OLD/Sp2013/5630/Jorge_Nocedal_Numerical_optimization_267490.pdf
+"""
+
+import jax
+import jax.numpy as np
```

Comment:
We're moving to importing ``jax.numpy as jnp`` in the rest of the package; please update for consistency.

Reply:
Done.